### PR TITLE
[EASY] Add missing tokens on mainnet and Gnosis Chain

### PR DESCRIPTION
**Pull request by harisang**
This PR adds some ERC-20 missing tokens on Ethereum Mainnet and Gnosis Chain.

### CODE REVIEW

**Comment by jeff-dude** [comment]
@harisang i assume with this large list of adds, you are still using an automated approach to finding these? do you happen to know if these tokens are all recently deployed, and it's possible our automated source will write them for us soon?

**Comment by harisang** [comment]
> @harisang i assume with this large list of adds, you are still using an automated approach to finding these? do you happen to know if these tokens are all recently deployed, and it's possible our automated source will write them for us soon?

Indeed, i am using an automated script that can be found here: https://github.com/cowprotocol/data-misc

Tbh i haven't caught up with dune's automated way of picking tokens, and i admit i hadn't seen such an alert on our end for some time. But given that we got an alert, i decided to do a PR since it is very easy to do so with the script @bh2smith had developed.

File: dbt_subprojects/tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql [hunk 1]
```diff
@@ -4587,4 +4587,433 @@ FROM (VALUES
...
+    , (0xf858184f5f04ab02e787dceadb00b04ab96ef54e, 'KIWI', 18)
+    , (0x69420ea9361a7bbb61187a5d05b7d44a0b0002be, 'MUMU', 18)
+    , (0x8353157092ed8be69a9df8f95af097bbf33cb2af, 'GHO/USDT/USDC', 18)
+    , (0x00364c6d90727d415225f863b98500729ee2ed2e, 'MEGAX', 18)
+    , (0x901a26110a1a4df328c26ec5779a4d9203df4c9a, 'RMUSK', 18)
+    , (0x21a4d9a037a06b0dfe7d6067edbe10b64795dee6, 'BDAG', 18)
+    , (0x09db87a538bd693e9d08544577d5ccfaa6373a48, 'ynETH', 18)
+    , (0xddaf27167929cd045a7d97d09a4fa1046ece3d89, 'MELLSTROY', 18)
+    , (0x3ad4f189f08cdc60496eebb3bd70b90df28b7455, 'NEIRO', 18)
+) AS temp_table (contract_address, symbol, decimals)
```
Comment (bh2smith) [inline dbt_subprojects/tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql:5019]:
```suggestion
) AS temp_table (contract_address, symbol, decimals)

```

File: dbt_subprojects/tokens/models/tokens/gnosis/tokens_gnosis_erc20.sql [hunk 2]
```diff
@@ -7798,4 +7798,9 @@ FROM (VALUES
     , ( 0x4e5af8c1bd8ec1ffafa8a6043e57c3d384a7c574, 'ZTLN', 18)
     , ( 0x0deee1f9b96cac0b0ec85e8f800bc0b33e901e9a, 'IDOL', 18)
     , ( 0xfef5f9d8fe3b23fe4a242a461e537bb69364de94, 'VCUSD', 18)
-) AS temp_table (contract_address, symbol, decimals)
+    , (0xdd439304a77f54b1f7854751ac1169b279591ef7, 'EURe/sDAI', 18)
+    , (0xedbc7449a9b594ca4e053d9737ec5dc4cbccbfb2, 'aGnoEURe', 18)
+    , (0x06135a9ae830476d3a941bae9010b63732a055f4, 'stEUR/EURe', 18)
+    , (0xc791240d1f2def5938e2031364ff4ed887133c3d, 'rETH', 18)
+    , (0xf490c80aae5f2616d3e3bda2483e30c4cb21d1a0, 'osGNO', 18)
+) AS temp_table (contract_address, symbol, decimals)
```
Comment (bh2smith) [inline dbt_subprojects/tokens/models/tokens/gnosis/tokens_gnosis_erc20.sql:7806]:
```suggestion
) AS temp_table (contract_address, symbol, decimals)

```

I guess the script doesn't add new lines :(

**Review by bh2smith — APPROVED** [review]

**Comment by jeff-dude** [comment]
thanks for the clarification. do you mind running the script again, to see if some tokens drop out (assuming our automated source picked them up in the last few days)?

for our automated source, we are just reading a third party vendor API and loading to a table on dune. spellbook simply reads from that new table, then unions these manual files for any missing in automated.

my hope is that if we give it a few days, most tokens would be accounted for by the automated source we're using. we are aware of some lack of coverage, but also aware of adding tokens "too soon" on the manual files

**Pull request closed without merging**